Source/Core/InputCommon/GCPadStatus.h

```diff
@@ -48,7 +40,7 @@ struct GCPadStatus
   u8 triggerRight;  // 0 <= triggerRight <= 255
   u8 analogA;       // 0 <= analogA      <= 255
   u8 analogB;       // 0 <= analogB      <= 255
-  s8 err;           // one of PAD_ERR_* number
+  bool isConnected{true};
```

Comment:
What's the rational for making this true by default? If an arbitrary freestanding status struct is created, would it not be more logical to represent disconnected by default?